docs/source/reference/numpysupported.rst

```diff
@@ -182,6 +182,7 @@ floating-point and complex numbers:
 * :func:`numpy.linalg.eigvalsh` (only the first argument).
 * :func:`numpy.linalg.inv`
 * :func:`numpy.linalg.lstsq`
+* :func:`numpy.linalg.matrix_power`
```

Comment:
Something here perhaps? http://numba.pydata.org/numba-doc/dev/reference/fpsemantics.html#linear-algebra
Or we could go for a new entry specifically discussing `linalg` internals?